File: 2624-difference-between-element-sum-and-digit-sum-of-an-array/difference-between-element-sum-and-digit-sum-of-an-array.java
class Solution {
    public int differenceOfSum(int[] nums) {
    int elementSum = 0;
        int digitSum = 0;
        for (int num : nums) {
            elementSum += num;
            String s = String.valueOf(num);
            for (char c : s.toCharArray()) {
                digitSum += c - '0';
            }
        }
        return Math.abs(elementSum - digitSum);     
    }
}

File: 2624-difference-between-element-sum-and-digit-sum-of-an-array/difference-between-element-sum-and-digit-sum-of-an-array.cpp
class Solution {
public:
    int differenceOfSum(vector<int>& nums) {
        int esum=0;
        int dsum=0;
        for(auto s:nums)
        {
            esum+=s;
        }
        for(auto s:nums)
        while(s){
            int x=s%10;
            dsum+=x;
            s=s/10;
        }
        return esum-dsum;
    }
};